ImageFeed/Services/DateService.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  DateService.swift
+//  ImageFeed
+//
+//  Created by Евгений on 07.05.2023.
+//
+
+import Foundation
+
+final class DateService {
+    let dateFormatter = DateFormatter()
+    let isoDateFormatter = ISO8601DateFormatter()
+    
+    func dateToString(date: Date?) -> String {
+        guard let date = date else { return ""}
+        dateFormatter.dateStyle = .medium
+        dateFormatter.timeStyle = .none
+        let string = dateFormatter.string(from: date)
+        return string
+    }
+    
+    func toDate(_ string: String) -> Date? {
+        let date = isoDateFormatter.date(from: string)
+        return date
```

Comment:
в данном случае можно еще короче
```suggestion
        return isoDateFormatter.date(from: string)
```